ai-ml/jark-stack/terraform/helm-values/ingress-nginx-values.yaml

```diff
@@ -5,7 +5,7 @@ controller:
       service.beta.kubernetes.io/aws-load-balancer-type: external
       service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
       service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
-      service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+      service.beta.kubernetes.io/aws-load-balancer-scheme: internal # Private Load Balancer can only be accessed within the VPC
```

Comment:
This will break the blog. Why are we doing this? Can't we just add a comment?

Reply:
Can we change the blog to do port-forward instead of using LB?